ReadMe.md

```diff
@@ -16,8 +16,8 @@ Tool for easy ClickHouse backup and restore with cloud storages support
 - Works with AWS, GCS, Azure, Tencent COS, FTP, SFTP
 - **Support of Atomic Database Engine**
 - **Support of multi disks installations**
-- **Support for any custom remote storage like `rclone`, `kopia`, `restic`**
-- Support of incremental backups on remote storages
+- **Support for many custom remote storage like `rclone`, `kopia`, `restic`**
```

Comment:
no, you can define any custom storage which you can wrap with bash scripts and adopt

Reply:
It just reads a bit better: any means all. Does the solution support _all_ remote storage types that are out there?

Reply:
Yes, if you can wrap it via bash script and adapt it to using, you can
we need to say to user, you can adopt ANY extenral tool which could upload / download / list / delete